### DISCUSSION TITLE
Discussion about setting time of chromatin opening to 0

### DISCUSSION THREAD

**Discussion by PernilleYR** [welch-lab/MultiVelo#23] [General]
Hi Welch lab, 

Amazing work with MultiVelo, I am looking forward to soon apply it to my data!

I have a question related to the time of chromatin opening t0. In your manuscript it is explained that t0 = 0 for all genes. I might have misunderstood, however I interpreted that the inferred latent time axis is shared across all the genes (meaning that t=1 for gene A and t=1 for gene B represent the same time point in the latent time/differentiation stage, and as such that t=0 is the root, i.e where the stem cells are). I would imagine that the regulatory elements of some genes might stay close during the early stages of stem cell commitment, and would only start to open in more committed stages, and as such the chromatin would not yet be open in stem cells, t=0.   Did I misunderstood and mazbe each gene has its own latent time? Is it possible to capture genes for which the regulatory regions will only start to open later along the trajectory? Can we learn when they would start to open? 

Additional question that might or not be related, in several figures of your manuscript, showing change of c(t) over time such as Figure 5e. I was surprised of how much c follows the fitted line with no variance at all in the primed state, do you know what might be the reason? 

  Thank you! :)

**Comment by jw156605** [comment]
Thanks for the kind words! A few points of clarification:

  *   Each gene is fitted on its own separate time scale. t_o = 0 for all genes, because there is simply no way to infer the progression of time if c(t)=u(t)=s(t)=0. In other words, time may pass while cells sit in the stem cell state with completely closed chromatin for a particular gene, but we have no molecular signals of the passage of time, so we just have to collapse all of the cells to t=0 (for that gene). If we infer shared latent time post-hoc, we can use all genes to reconstruct the order of events.
  *   c(t) closely follows the fitted line during the priming phase because ordering cells by their value of c is the only way to infer time during the priming phase (since u(t) = s(t) = 0).
**********************************************************
Electronic Mail is not secure, may not be read every day, and should not be used for urgent or sensitive issues
